02LessonTwo/homework/Homework.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="header">
+    <h1>Larry Gutierrez</h1>
+    <h1>EST.1991<h1>
+  </div>
+  <div id="left">
+    <table>
+      <th>
+        <h2>About Me</h2>
+        <p>Whats up guys im Larry also know in the streets as Bird you guys can most probably make the
+        connection there. Lived all my life in San Antonio so you know im a spurs fan.
+        A entrepenuer at heart always curious about how thing come into existense, how they work
+        and grow. I like to think of my self as a discplined, honest and loving individual. Family
+        is everything to me. I love chinesse food like seriously i can probaly eat all the time.
+       </p>
+     </th>
+   </table>
```

Comment:
You used a `<table>` tag here but I think you could have gone without one and the content of the page would not have changed. The only difference I see when I removed the table tags is that your About Me heading and paragraph text are no longer centered but you could still achieve the desired look with CSS.